### PR TITLE
Add Linux Pip Nightly Build

### PR DESCRIPTION
**Pull request by hogepodge**
Adds a nightly build for Linux CPU to the pip build. This patch is built on the Docker image build, which uses manylinux2014 to build TVM then extracts the wheels. Not compatible with the latest release, so only builds nightly.

### CODE REVIEW

**Comment by hogepodge** [comment]
@tqchen @icemelon9 Please double check my work that I'm building and pushing the wheels properly. Is there anything else I need to do to enable this in the download matrix?
@leandron ping for review

**Comment by hogepodge** [comment]
Looks like the build failed because of the empty HASH value. I'll need a more robust solution

**Comment by hogepodge** [comment]
It looks like this won't build without an update to the package/cpu docker image.

@icemelon9 what are the steps to update this package? How safely can we do it without impacting the rest of the build?

https://hub.docker.com/r/tlcpack/package-cpu/tags?page=1&ordering=last_updated

**Comment by hogepodge** [comment]
Hold this PR for testing and image updating.

File: docker/repository.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hogepodge
```
Comment (hogepodge) [inline docker/repository.txt:1]: This should be changed to `tlcpack` once we're happy with testing.

File: .github/workflows/docker_cpu_nightly.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: '0 6 * * *' # 6 AM UTC
+
+jobs:
+  Build-Cpu:
```
Comment (hogepodge) [inline .github/workflows/docker_cpu_nightly.yaml:15]: All of the other jobs are named `Build`, is this for any particular reason. For local testing with `act`, it helps to give these different names. This either needs to be changed to make it consistent with the other build files, or the other build files should be changed to disambiguate them.

File: docker/version.txt [hunk 3]
```diff
@@ -1 +1 @@
-0.1
+prod
```
Comment (hogepodge) [inline docker/version.txt:1]: Before merging, all of the docker images would need new tags. Doing this allows us to define which images are being used for the build without locking in on a particular version number.

**Comment by tqchen** [comment]
Thanks @hogepodge. There are quite a few places that we need to update to make the workflow consistent with the current one. I created an updated PR based on this one https://github.com/tlc-pack/tlcpack/pull/47

**Comment by tqchen** [comment]
Thanks @hogepodge superseded by #47

**Pull request closed without merging**